docs/data_classes.rst

```diff
@@ -395,7 +404,8 @@ We can now define our `~sunraster.RasterSequence` by doing:
 .. code-block:: python
 
   >>> from sunraster import RasterSequence
-  >>> my_sequence = RasterSequence([raster0, raster1, raster2], meta=seq_meta, slit_step_axis=0)
+  >>> my_sequence = RasterSequence([raster0, raster1, raster2], meta=seq_meta, common_axis=0)
+  ['slit step' None 'spectral']
```

Comment:
This is caused by an erroneous print statement and should be fixed in the code rather than the docs.  So could you undo this change?

In fact, I'm almost certain this has been fixed in code in one of the PRs.

Reply:
This is caused by an erroneous print statement and should be fixed in the code rather than the docs.